README.md

```diff
@@ -88,23 +88,16 @@ This step will create an AAD Application and Service Principal in the specified
 
     Add an environment called `Infra-Test` with the following secrets:
 
-    - `AZURE_CREDENTIALS`: json containing the credentials of the service principal (outputted from the previous step) in the following format:
-
-        ```json
-        {
-            "clientId": "xxx",
-            "clientSecret": "xxx",
-            "tenantId": "xxx",
-            "subscriptionId": "xxx",
-            "resourceManagerEndpointUrl": "management.azure.com"
-        }
-        ```
-
+    - `ARM_CLIENT_ID`: Client ID of the service pricipal created in step 2
+    - `ARM_CLIENT_SECRET`: Client secret of the service pricipal created in step 2
+    - `ARM_TENANT_ID`: Tennant ID containing the Azure subscription to deploy into
+    - `ARM_SUBSCRIPTION_ID`: Subscription ID of the Azure subscription to deploy into
     - `SUFFIX`: Suffix used for naming resources. Must be unique to this repository e.g. `abcd`
     - `LOCATION`: Name of an Azure location e.g. `uksouth`. These can be listed with `az account list-locations -o table`
     - `ENVIRONMENT`: Name of the environment e.g. `dev`, also used to name resources
     - `DEVCONTAINER_ACR_NAME`: Name of the Azure Container Registry to use for the devcontainer build. This may or may not exist. e.g. `flowehrmgmtacr`
-    - `LOCAL_MODE`: Set to `false`
+    - `ORG_GH_TOKEN`: GitHub [PAT](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) with scopes to clone any repositories defined in `config.transform.yaml`. This may be added as a repository rather than envrionment secret and be resued betweeen envrionments
```

Comment:
nit: typo `reused`, and also thanks for adding this doc (I guess I should have done that!) 🙏